examples/idx_mapping_watcher/main.go

```diff
@@ -42,144 +46,143 @@ import (
 // HTTP and Log), and example plugin which demonstrates index mapping watcher functionality.
 func main() {
 	// Init close channel to stop the example
-	closeChannel := make(chan struct{}, 1)
+	exampleFinished := make(chan struct{}, 1)
 
-	f := vpp.Flavor{}
+	// Start Agent with ExampleFlavor (combinatioplugin.GoVppmux, n of ExamplePlugin & reused cn-infra plugins)
+	flavor := ExampleFlavor{closeChan: &exampleFinished}
```

Comment:
let's have closeChan reference only in plugin then

ExampleFlavor{ExamplePlugin:ExamplePlugin{closeChan: &exampleFinished{}